entries/accordion.xml

```diff
@@ -86,6 +86,21 @@
 				</desc>
 			</type>
 		</option>
+		<option name="classes" type="Object" default='Object' example-value='{ "ui-accordion-header": null }'>
```

Comment:
How's this?

![...](http://bassistance.de/i/a10a00.png)

The styling for `<example>` element doesn't match the one for the generated examples. I guess one more TODO for grunt-jquery-content to add to the list.